pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -62,7 +62,7 @@ func (q *graceTerminateRSList) add(rs *listItem) bool {
 		return false
 	}
 
-	klog.V(5).Infof("Adding rs %v to graceful delete rsList", rs)
+	klog.V(5).InfoS("Adding rs to graceful delete rsList", "rs", fmt.Sprintf("%v", rs))
```

Comment:
rs doesn´t seems very meaninful, but I´m not familiar with ipvs enough to judge or give an alternative, @andrewsykim what do you think?